### PR TITLE
Add documentation on running tests

### PR DESCRIPTION
**Pull request by jonsherrard**
Added a note to the contributing docs on running the test for fellow lerna newbies like me.

### CODE REVIEW

File: contributing.md [hunk 1]
```diff
@@ -62,6 +63,13 @@ Code contributions are very welcome.  It’s often good to first create an issue
 to report a bug or suggest a new feature before creating a pull request to
 prevent you from doing unnecessary work.
 
+## Running the tests
+
+1.  `npm/yarn install`
+2.  `yarn lerna exec yarn install`
```
Comment (timneutkens) [inline contributing.md]: `lerna bootstrap` is supposed to take care of this 🤔

Comment (jonsherrard) [reply contributing.md]: This issue is referenced in the travis.yml where `lerna bootstrap` has been . replaced with script 2:  https://github.com/lerna/lerna/issues/1457

Maybe I could change the instructions to say "lerna boostrap _should_ work", but if it doesn't, try: 'yarn lerna exec yarn install'" ?

Comment (timneutkens) [reply contributing.md]: I wonder if the alternative is yarn workspaces, this is what we do for Next.js, and just recommend everyone uses `yarn`

File: contributing.md [hunk 2]
```diff
@@ -62,6 +63,13 @@ Code contributions are very welcome.  It’s often good to first create an issue
 to report a bug or suggest a new feature before creating a pull request to
 prevent you from doing unnecessary work.
 
+## Running the tests
+
+1.  `npm/yarn install`
```
Comment (johno) [inline contributing.md]: We should probably prefer one dep manager and update contrib docs to reflect that. Perhaps here we should prefer `yarn`?

```suggestion
1.  `yarn`
```

File: contributing.md (hunk 1 above)
Comment (johno) [reply contributing.md]: I wonder if this should be moved to a troubleshooting section at the bottom of this doc?

**Comment by johno** [comment]
Thanks!

**Pull request merged**